### PR TITLE
Startup args for r2modman

### PR DESCRIPTION
**Pull request by emma-miler**
Autogenerate `ns_startup_args.txt` and add flag to bypass `ns_startup_args.txt`

### CODE REVIEW

File: LauncherInjector/main.cpp [hunk 1]
```diff
@@ -312,6 +312,21 @@ int main(int argc, char* argv[])
...
 		PrependPath();
 
+		if (!fs::exists("ns_startup_args.txt"))
+		{
+			std::ofstream file("ns_startup_args.txt");
+			std::string defaultArgs = "-multiple -novid -noborder -windowed";
+			file.write(defaultArgs.c_str(), defaultArgs.length());
+			file.close();
+		}
+		if (!fs::exists("ns_startup_args.txt"))
```
Comment (laundmo) [inline LauncherInjector/main.cpp:322]:
```suggestion
		if (!fs::exists("ns_startup_args_dedi.txt"))
```

**Review by laundmo — CHANGES_REQUESTED** [review]

File: LauncherInjector/main.cpp [hunk 2]
```diff
@@ -312,6 +312,21 @@ int main(int argc, char* argv[])
 	{
 		PrependPath();
 
+		if (!fs::exists("ns_startup_args.txt"))
+		{
+			std::ofstream file("ns_startup_args.txt");
+			std::string defaultArgs = "-multiple -novid -noborder -windowed";
```
[the remark this answers is not in the document]
Comment (emma-miler) [reply LauncherInjector/main.cpp:318]: Oops, i forgot i modified my file. I just copied the stuff i had in mine and didnt think twice. Im not at my PC right now, so if you could PR the vanilla file onto my branch ill merge it. Ping me on discord if you do ^^

[the remark this answers is not in the document]
Comment (GeckoEidechse) [reply LauncherInjector/main.cpp:318]: It's
- `-multiple` for [`ns_startup_args.txt`](https://github.com/R2Northstar/Northstar/blob/main/release/ns_startup_args.txt)
- `+setplaylist private_match` for [`ns_startup_args_dedi.txt`](https://github.com/R2Northstar/Northstar/blob/main/release/ns_startup_args_dedi.txt)

^^

[the remark this answers is not in the document]
Comment (pg9182) [reply LauncherInjector/main.cpp:318]: About `+setplaylist private_match`, I actually think we should.consider making it set it if it's not provided by default if there aren't any downsides.

**Review by abarichello — CHANGES_REQUESTED** [review]
Format check is failing https://github.com/R2Northstar/NorthstarLauncher/runs/5363501103?check_suite_focus=true

**Review by abarichello — APPROVED** [review]
Thanks, can you also add the new startup argument to the wiki? (targeting next-release branch) https://r2northstar.gitbook.io/r2northstar-wiki/using-northstar/launch-arguments

**Pull request merged**